Student/Student/Info.plist

```diff
@@ -102,6 +102,8 @@
 		<string>com.instructure.icanvas.watchApp</string>
 		<string>com.instructure.siri.shortcut.getgrades</string>
 	</array>
+	<key>UIBackgroundModes</key>
+	<array/>
```

Comment:
Enabling Background Modes. Something we should do since we use a background URLSession.